mgr/Conscript-standard

```diff
@@ -197,7 +197,8 @@ if ( $pkg eq "Jevp") {
 	      .   $main::PATH_SEPARATOR . "#OnlTools/Jevp/StJevpBuilders"
 	      .   $main::PATH_SEPARATOR . "#OnlTools/Jevp/StJevpViewer"
 	      .   $main::PATH_SEPARATOR .  $CPPPATH;
-    $CPPFLAGS .= " -DQT_THREAD_SUPPORT -DQT_SHARED -DQT_NO_DEBUG ";
+    $CPPFLAGS .= " -DQT_THREAD_SUPPORT -DQT_SHARED -DQT_NO_DEBUG -DRTS_ENABLE_LOG ";
```

Comment:
This was a topic of brief discussion in PR #673 ...which was left with the open question of whether there was an alternative for specifying the RTS_ENABLE_LOG flag at compile-time, as opposed to enabling it for all compilations (albeit only for the Jevp package). I have nothing to add to this review other than to say it would be good to get an answer to that question.